images/macos/provision/core/powershell.sh

```diff
@@ -2,7 +2,16 @@
 source ~/utils/utils.sh
 
 echo Installing PowerShell...
-brew install --cask powershell
+psRelease=$(curl -s "https://api.github.com/repos/PowerShell/PowerShell/releases/latest")
+psDownloadUrl=$(echo $psRelease | jq -r '.assets[].browser_download_url | select(contains("osx-x64.pkg"))' | head -n 1)
```

Comment:
I think we can make a helper as it was done for the windows here https://github.com/actions/virtual-environments/pull/4645/files
Or is it too complicated for bash?

Reply:
Created [an issue](https://github.com/actions/virtual-environments-internal/issues/3139) for that